ambry-rest/src/main/java/com.github.ambry.rest/NettyServerFactory.java

```diff
@@ -74,27 +73,7 @@ protected void initChannel(SocketChannel ch) {
               .addLast("processor", new NettyMessageProcessor(nettyMetrics, nettyConfig, requestHandler));
         }
       };
-    } else {
```

Comment:
I have a generic null argument check (see lines above).

I removed the specific messaging because @pnarayanan had comments long ago that it was code bloat and I agree with him.